src/main/java/com/school/project/service/impl/UserServiceImpl.java

```diff
@@ -32,8 +36,11 @@ public User create(User user) {
     }
 
     @Override
-    public void update(UserDto user, Long id) {
-        User userById = getUserById(id);
+    public void update(User user, Long id) {
+        User userById;
+        Optional<User> byId = userRepository.findById(id);
+        if (!byId.isPresent()) throw new UserNotFoundException("User with id :" + id + "is not found");
+        else userById = byId.get();
         userById.setFirstName(user.getFirstName());
         userById.setLastName(user.getLastName());
         userById.setBirthDate(user.getBirthDate());
```

Comment:
`userById.setFirstName(user.getFirstName());
        userById.setLastName(user.getLastName());
        userById.setBirthDate(user.getBirthDate());
`
think about how to write it in one line